src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -2226,25 +2226,7 @@ void MulticopterPositionControl::control_auto(float dt)
 
 			_pos_sp = pos_sp;
 
-		} else if (_pos_sp_triplet.current.type == position_setpoint_s::SETPOINT_TYPE_VELOCITY) {
```

Comment:
Should we remove `SETPOINT_TYPE_VELOCITY` from the `position_setpoint` uORB message, since it's not used?